Modules/Server/Sources/PocketCastsServer/Public/StatsManager.swift

```diff
@@ -29,6 +29,39 @@ public class StatsManager {
         }
     }
 
+    func updateStatsIfNeeded(savedDynamicSpeed: TimeInterval, savedVariableSpeed: TimeInterval, totalListenedTo: TimeInterval, totalSkipped: TimeInterval, savedAutoSkipping: TimeInterval) {
+        let minimumStatsChangeToUpdate = 100
```

Comment:
I'm debating whether this should be `100` or something bigger or smaller.

Maybe if the local stat is just smaller than remote we should update?